native/android/build.gradle

```diff
@@ -7,10 +7,30 @@ buildscript {
         compileSdkVersion = 29
         targetSdkVersion = 29
         kotlinVersion = "1.3.50" // TODO: Remove in NATIVE-490
+        // config for MapsLibre
+        rnmbglMapboxLibs = {
+            implementation("org.maplibre.gl:android-sdk:9.2.1")
+            implementation("com.mapbox.mapboxsdk:mapbox-sdk-turf:5.3.0")
+        }
+        rnmbglMapboxPlugins = {
+            implementation("com.mapbox.mapboxsdk:mapbox-android-gestures:0.7.0")
+            implementation("com.mapbox.mapboxsdk:mapbox-android-plugin-localization-v9:0.12.0") {
+                exclude group: 'com.mapbox.mapboxsdk', module: 'mapbox-android-sdk'
+            }
+            implementation("com.mapbox.mapboxsdk:mapbox-android-plugin-annotation-v9:0.8.0") {
+                exclude group: 'com.mapbox.mapboxsdk', module: 'mapbox-android-sdk'
+            }
+            implementation("com.mapbox.mapboxsdk:mapbox-android-plugin-markerview-v9:0.4.0") {
+                exclude group: 'com.mapbox.mapboxsdk', module: 'mapbox-android-sdk'
+            }
+        }
```

Comment:
I suppose we are using maplibre on android right? to be extra cautious we can add these lines here:

https://github.com/ehrenamtskarte/ehrenamtskarte/blob/main/frontend/android/app/build.gradle#L65

This remove the only proprietrary dependency of mapbox 9 on android. Maybe it is still included in some plugin. If it still compiles with this change then it would be nice to have.

Reply:
I tried to integrate it, but i dunno how to achieve that into our build.cradle file structure. Maybe you can give it a try @maxammann

Reply:
ok i found the correct build.gradle file ;)